AK/StringView.cpp

```diff
@@ -178,6 +178,26 @@ bool StringView::contains(char needle) const
     return false;
 }
 
+bool StringView::contains(const StringView& needle) const
+{
+    if (needle.length() > length())
+        return false;
+
+    if (needle.is_empty())
+        return true;
+
+    if (needle.length() == 1)
+        return contains(needle[0]);
+
+    auto length_diff = length() - needle.length();
+    for (size_t i = 0; i < length_diff; ++i) {
+        if (memcmp(m_characters + i, needle.m_characters, needle.length()) == 0)
+            return true;
+    }
```

Comment:
Perhaps we could use `memmem()` here instead. Not that we have `memmem()` yet but it would match `String::contains()` a bit better :)

Reply:
so..."implement memmem" is what you're saying?

Reply:
Eh, I was just thinking out loud. But sure, it would be nice to have `memmem`!